Wireframe/index.html

```diff
@@ -4,10 +4,62 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <meta name="description" content="Get to know what is Git.">
+    <link rel="stylesheet" href="css/style.css">
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    <header>
+      <h1> Get to know Git</h1>
+    </header>
+    <main class="main">
+<!-- What is Git? -->
+      <section class="introduction">
```

Comment:
This section lacks a heading. Please consider using headings in each section. You can learn more about heading here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements